controllers/enrollmentkeys.go

```diff
@@ -148,6 +207,7 @@ func createEnrollmentKey(w http.ResponseWriter, r *http.Request) {
 //			Responses:
 //				200: RegisterResponse
 func handleHostRegister(w http.ResponseWriter, r *http.Request) {
+	shouldOmitRelaying := r.URL.Query().Get("omitRelay") == "true"
```

Comment:
if enrollment key has a relay attached it, it should be automatically relayed, remove query param